index.js

```diff
@@ -90,18 +92,25 @@ module.exports = {
     });
   },
 
+  fileMapTokens(options) {
+    let { addonInfo, testAppInfo } = options.locals;
+
+    return {
+      __addonLocation__: () => addonInfo.location,
+      __testAppLocation__: () => testAppInfo.location,
+    }
+  },
+
   locals(options) {
-    let entity = { name: 'dummy' };
+    let entity = { name: 'my-addon' };
```

Comment:
Looking at this again, I think `my-addon` is just as wrong as `dummy` here. It was a leftover from copying the original blueprint, see https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/index.js#L123-L131. The first block of vars (those without the addon* prefix) probably relate to the dummy app. As we don't have to deal with the dummy/test-app as part of this blueprint, I think we should remove all of those (and possibly fix accidental use of these in files). 

As this is not really related to your changes here, we/I can just as well clean this up in a follow up PR!

Reply:
Sound's good to me!